index.html

```diff
@@ -3,17 +3,117 @@
   <![endif]-->
 </head>
 
-<body>
-<!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
-</body>
-</html>
\ No newline at end of file
+  <body>
+   <div class="container">
+
+    
+        <nav class="navigation">
+        <div class="navi">
+          <a href="services.html">Services</a>
+          <a href="#">Product</a>
+          <a href="#">Vision</a>
+          <a href="#">Features</a>
+          <a href="#">About</a>
+          <a href="#">Contact</a>
+        </div>
+          <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+        </nav>
+      
+          
+      
+    
+    
+    <div class="headtwo">
+      <section class="leftpic">
+        <h1>Innovation<br/>
+          <p>On<br>
+          Demand</p>
+          
+        </h1>
+        <button>Get Started</button>
+         </section>
+         <aside class="rightpic">
+        <img src="img/header-img.png" alt="Image of a code snippet.">
+      </aside>
+      
+    </div>
+  
+    <div class="mid">
+       
+        <section class="a">
+        <h4>Features</h4>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+        Suspendisse
+        eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </section>
+        
+       
+      <section class="a">
+        <h4>About</h4>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+        Suspendisse
+        eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </section>
+        </div>
+  
+    <div>
+      <img class="c" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+    </div>
+    <div class="mid">
+        <section class="a">
+        <h4>Services</h4>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+        Suspendisse
+        eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </section>
+      
+        <section class="a">
+        <h4>Product</h4>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+        Suspendisse
+        eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </section>
+
+        <section class="a">
+        <h4>Vision</h4>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+        Suspendisse
+        eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </section>
+    </div>
+    
+    
+    <footer>
+      <h4>Contact</h4><br>
+      <p>123 Way 456 Street<br>
+      Somewhere, USA</p><br>
+      <p>1 (888) 888-8888</p><br>
+      sales@greatidea.io<br>
+      
```

Comment:
```suggestion
    <p>123 Way 456 Street<br>
      Somewhere, USA</p><br>
      <p>1 (888) 888-8888</p><br>
      sales@greatidea.io<br>
```
Consider wrapping this section in a <address> tag for more semantic HTML, plus help with search engine optimization.

```suggestion
<address>
    <p>123 Way 456 Street<br>
      Somewhere, USA</p><br>
      <p>1 (888) 888-8888</p><br>
      sales@greatidea.io<br>
      </address>
```